### ISSUE TITLE
Relax is build on Node.js version?

### ISSUE THREAD

**Issue by vanloc0301** [relax/relax#308]
I think you should add Relax is build on Node.js version x.x.

Because I have many problem when install latest Node.js version and tried to install Relax.

Each version Relax, I have error when try run command: `npm install`.

I install on version: 

> node v6.7.0
> npm  v3.10.8

Today, I meet error code:

`Unsupported platform for fsevents@1.0.14: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})`

when try run: `npm install`

And error code:

`
relax@0.0.0 dev: `parallelshell "npm run webpackServer" "npm run watchServer"`
Failed at the relax@0.0.0 dev script 'parallelshell "npm run webpackServer" "npm run watchServer"'.
Make sure you have the latest version of node.js and npm installed.
If you do, this is most likely a problem with the relax package,
`

I also install `Webpack`.

Can anyone tell me how to fix this error?

Thanks.

**Comment by bruno12mota** [comment]
Hey! Use node v4 or node v5 and it should install without errors

**Comment by vanloc0301** [comment]
I also tried with Node v5.11.0 but it show error similar.

**Comment by bruno12mota** [comment]
The error seems to be with sharp dependency. Are you using windows? Check if you have all the dependencies needed http://sharp.dimens.io/en/stable/install

**Comment by vanloc0301** [comment]
Yes. I using Windows 10 x64.

> Windows x64
> 
> Windows x64 Build Status
> 
> libvips and its dependencies are fetched and stored within node_modules\sharp during npm install. This involves an automated HTTPS download of approximately 9MB.
> 
> Only 64-bit (x64) node.exe is supported.

This mean I only install Node.js and run command: `npm install`?

I was run and appear error.

**Comment by bruno12mota** [comment]
Try downgrading npm to 3.10.7 https://github.com/npm/npm/issues/14042. If the error persists let me know

**Comment by vanloc0301** [comment]
Thanks @bruno12mota 
Install 3.10.7 will resolve the problem with sharp.

But other error appear.

`Failed at the fsevents@1.0.14 install script 'node-pre-gyp install --fallback-to-build'.`

I installed `npm install node-pre-gyp` before run command `npm install`.

**Comment by bruno12mota** [comment]
Have you tried `npm install -g node-gyp`?

**Comment by vanloc0301** [comment]
Yes. I also installed node-gyp and node-pre-gyp. This error still appear.
